### DISCUSSION TITLE
Is it possible to filter on a subobject field not null? (answered)

### DISCUSSION THREAD

**Discussion by rasberrylasagna** [prisma/prisma#9694] [Q&A]
Hey guys, I sure I'm missing something obvious, but is it possible to do a query like this?

```ts
const res = await prisma.user.findMany({
    where: {
      post: {
        url: {
          not: null,
        },
      },
    },
  })
```

That exact query does not work, nor does replacing null with undefined. Also wrapping the field itself in a larger NOT does not work.

Thanks for your help!

EDIT: It was a silly mistake, the error message was a bit misleading but I realized that said field was marked required in the schema. 😓 
This can be deleted / marked answered. Thanks!

**Comment by ryands17** [comment]
@rasberrylasagna 👋 

Can you share your `schema.prisma`?

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
